packages/cursorless-engine/src/util/setSelectionsAndFocusEditor.ts

```diff
@@ -20,13 +20,15 @@ export async function setSelectionsAndFocusEditor(
   await editor.focus();
 }
 
-export function setSelectionsWithoutFocusingEditor(
+export async function setSelectionsWithoutFocusingEditor(
```

Comment:
worth checking to make sure this function is awaited everywhere it is called; I noticed you missed a couple `await`s on `setSelections`

Reply:
Checked fine